agenthub/codeact_agent/codeact_agent.py

```diff
@@ -221,7 +221,7 @@ def step(self, state: State) -> Action:
                 f'\n\nENVIRONMENT REMINDER: You have {state.max_iterations - state.iteration} turns left to complete the task.'
             )
 
-        response = self.llm.do_completion(
+        response = self.llm.completion(
```

Comment:
Do you know why this call is overwriting the `temperature` with 0 instead of passing on the value in the wrapper?
If it's a legacy line - before the LLMConfig introduction etc. - then that param should be removed, right?

Reply:
I don't know. I think this is out of scope in this PR, however. It might be a good idea to make a separate issue where we can look into it and discuss it on its merits? It's there for a long time, I'm fairly sure.

Reply:
Fair, agreed.